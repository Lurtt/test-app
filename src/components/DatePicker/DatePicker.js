import React from 'react'

export default function DatePicker(props) {
  return <input type="date" {...props} />
}
