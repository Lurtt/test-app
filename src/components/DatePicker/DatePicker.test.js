import React from 'react'
import renderer from 'react-test-renderer'

import DatePicker from './'

describe('<DatePicker />', () => {
  test('Should render', () => {
    const result = renderer.create(<DatePicker />).toJSON()
    expect(result).toMatchSnapshot()
  })

  test('Should accept {...props}', () => {
    const result = renderer
      .create(<DatePicker id={1} value="2007-05-16" />)
      .toJSON()
    expect(result).toMatchSnapshot()
  })
})
