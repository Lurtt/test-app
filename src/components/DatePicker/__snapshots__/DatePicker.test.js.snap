// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DatePicker /> Should accept {...props} 1`] = `
<input
  className="sc-bdVaJa YQfmf"
  id={1}
  type="date"
  value="2007-05-16"
/>
`;

exports[`<DatePicker /> Should render 1`] = `
<input
  className="sc-bdVaJa YQfmf"
  type="date"
/>
`;
